### ISSUE TITLE
map reduce less frequently

### ISSUE THREAD

**Issue by gboruk** [wilkerlucio/mongoid_taggable#22]
Is it possible to have an option to periodically (on a configurable period) to run the map reduce instead of running it on every document save?

**Comment by wilkerlucio** [comment]
interesting idea, I started some changes on project to make it possible, provably push something this week :)

**Comment by gboruk** [comment]
awesome, looking forward to it! thanks!

**Comment by yordis** [comment]
@wilkerlucio is this done?

**Comment by wilkerlucio** [comment]
@yordis sorry, I've not being working on this project for quite a while so I'm not sure

I see this commit here: https://github.com/wilkerlucio/mongoid_taggable/commit/fa74e02e7a01a38ddaab4380c35e3a22f5ff458a

which seems an improvement, maybe @astjohn or @calebclark are more able to answer that for you

**Comment by yordis** [comment]
@wilkerlucio will you be active on this. I don't want to re-write the same problem solution and this seems to be very good but I dont want to use something that you don't want to maintain at least talking PR

**Comment by wilkerlucio** [comment]
@yordis nope, I'm not active on this project anymore, a few years ago I added other people as collaborators (the fellows that I mentioned before) and they did some work then, but its also up to then.

I can still get pull requests, but if you really plan to do a lot of work I would suggest you to just rewrite, this library is pretty small after all ;)

**Comment by gboruk** [comment]
This one's a bit old. Is it closeable? :)
